### PR TITLE
checkout: fix module test

### PR DESCRIPTION
**Pull request by jeinfeldt**
Tests are failing due to missing config values

### CODE REVIEW

File: checkout/module_test.go [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
 )
 
 func TestModule_Configure(t *testing.T) {
-	if err := dingo.TryModule(new(checkout.Module)); err != nil {
+	cfgModule := &config.Module{
+		Map: new(checkout.Module).DefaultConfig(),
+	}
+
+	cfgModule.Map["session.backend"] = ""
+	cfgModule.Map["oauth.useFake"] = true
```
Comment (danielpoe) [inline checkout/module_test.go]: only fixable that way?
Feels a bit strange that the checkout modul needs to know this internals

Comment (jeinfeldt) [reply checkout/module_test.go]: This is due to the fact that the checkout module depends on the cart and the cart module depends on the oauth module from core (which relies on these flags). I am unsure how to resolve these dependencies correctly in the module test without providing these flags

I think the only other option is to merge all config maps of the dependent modules together.

**Comment by thebod** [comment]
Hi, I'm sorry, i accidentially pushed that fix to master.
This will be solved/cleaned up with the improved flamingo core config.

I'll close this, sorry!

File: checkout/module_test.go (hunk 1 above)
Comment (thebod) [reply checkout/module_test.go]: Problem is that the oauth module is not working "out-of-the-box". That's why the config needs to be here :(

**Pull request closed without merging**